cfme/infrastructure/pxe.py

```diff
@@ -945,12 +967,13 @@ def get_template_from_config(template_config_name):
                                  replacements=template_config['replacements'])
 
     script_data = script_data.read()
-
-    return CustomizationTemplate(name=template_config['name'],
-                                 description=template_config['description'],
-                                 image_type=template_config['image_type'],
-                                 script_type=template_config['script_type'],
-                                 script_data=script_data)
+    appliance = get_or_create_current_appliance()
```

Comment:
**Required**

Get appliance through the CFMENavigateStep appliance property.

Reply:
@mshriver: 
def get_template_from_config() is a separate method, it's not part of navigator class or you meant something else?
I have discussed this issue with @RonnyPfannschmidt and we have come to this conclusion that we don't have any other way as of now to handle this issue.

Reply:
Ha! My bad @anurag03 you're right, I read the diff wrong when I expanded it.

https://sadtrombone.com/